yt/docker/ytsaurus/Dockerfile

```diff
@@ -228,3 +258,75 @@ FROM scratch AS ytsaurus-server-override
 COPY --from=ytsaurus-server-override-bloated / /
 
 ##########################################################################################
+
+FROM python-base as odin-installer
+
+ARG ODIN_ROOT
+ARG ODIN_VIRTUAL_ENV
+ARG ODIN_CHECKS_DIR
+ARG ODIN_CHECKS_DATA_DIR
+
+ARG SOURCE_ROOT="${ODIN_ROOT}/ytsaurus"
+ARG ODIN_SOURCE_ROOT="${SOURCE_ROOT}/yt/odin"
+ARG BUILD_ROOT="${ODIN_ROOT}/build"
+ARG PYTHON_ROOT="${ODIN_ROOT}/python"
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
```

Comment:
Why `DEBIAN_FRONTEND=noninteractive` is omitted here?

Reply:
No good reason. I was copy and pasting from old Odin dockerfile. Added it to all apt-get install commands in this file.